.eslintrc.json

```diff
@@ -19,6 +19,7 @@
     "no-shadow-restricted-names": 2,
     "no-undef-init": 2,
     "no-undefined": 2,
+    "no-use-before-define": 2,
```

Comment:
Well if it works with all the Papa structure then fine, you can test that. But I'm pretty sure it was removed for this reason.